ci/coverity_success.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -uex
+
+rm -rf "_build.external${arch:-}"
```

Comment:
Isn't this effectively:
```suggestion
rm -rf "_build.external${arch}"
```
because `arch` is never unbound as it's *always* set in the calling environment.

Reply:
We don't use arch any more anywhere to the best of my knowledge, so we should just drop it.